package.json

```diff
@@ -76,7 +76,8 @@
     "**/@babel/traverse": "^7.21.2",
     "**/@babel/types": "^7.21.2",
     "**/@types/babel__generator/@babel/types": "^7.21.0",
-    "**/@types/node": "16.11.7",
+    "**/@types/node": "18.17.1",
+    "**/@types/react": "^16.14.25",
```

Comment:
For me locally, enzyme resolved to a different version, and this caused a type error. Maybe it's not necessary ultimately, once the dependencies are properly locked in the `yarn.lock`